scripts/generate_export.py

```diff
@@ -481,15 +502,14 @@ def sequences_overlap(true_seq, pred_seq):
 
     exception_filenames_df = pd.DataFrame(exception_filenames, columns=["File Name"])
     exceptions_df = pd.concat([exception_filenames_df, exception_ids_df], axis=1)
-    # for es in exception_submissions:
-    #     sub_job = INDICO_CLIENT.call(SubmissionResult(es.id, wait=True))
-    #     result = INDICO_CLIENT.call(RetrieveStorageObject(sub_job.result))
-    #     exceptions_revID.append(result.get("reviewer_id"))
+    for es in exception_submissions:
+        sub_job = IndicoClient.call(SubmissionResult(es.id, wait=True))
```

Comment:
It makes sense to get submission objects here but switch to using the indico_wrapper.